### ISSUE TITLE
Android app crash in few device 

### ISSUE THREAD

**Issue by mitul-neosoft** [transistorsoft/cordova-background-geolocation-lt#1438]
## Your Environment
* Plugin version: 4.15.2
* Platform: Android
* OS version: 13/14
* Device manufacturer / model: samsung p3s, samsung a34x, Xiaomi ziyi, Xiaomi zeus, Redmi merlin, samsung y2s, samsung a54x
* Cordova version (`cordova -v`):  12
* Cordova platform version (`cordova platform ls`): 12
* Plugin config provided to `#ready`:
```javascript <!-- syntax-highlighting:  paste your code below -->
                                  locationAuthorizationRequest: 'Always',
				// for Android
				backgroundPermissionRationale: {
					title: "Allow access to this device's location in the background?",
					message: "",
					positiveAction: "Change to Allow all the time",
					negativeAction: "Cancel"
				},

				reset: true,
				debug: false,
				stopOnTerminate: false, 
				stationaryRadius: 150,
				startOnBoot: true, 
				allowIdenticalLocations: true,
				desiredAccuracy: this.window.BackgroundGeolocation.DESIRED_ACCURACY_HIGH,
				foregroundService: true,
				geofenceProximityRadius: 3500,
				notification: {
					title: this.filter('translate')('xyz'),
					text: this.filter('translate')('Location service activated'),
					smallIcon: "drawable/fcm_push_icon"
				},
				distanceFilter: 150, 
				maxDaysToPersist: 1,
				url: `xyz`,
				method: 'POST'
```

## Expected Behavior
plugin should not crash app

## Actual Behavior
app crash when geofence add

## Steps to Reproduce
1. Start App and start adding geofence once we have permissions.

## Context
I was not able to reproduce in local but there is log on play store.

## Debug logs

<details>
	<summary>Logs</summary>

```<!-- Syntax highlighting:  DO NOT REMOVE -->
Exception java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.transistorsoft.locationmanager.geofence.TSGeofence.isPolygon()' on a null object reference
  at com.transistorsoft.locationmanager.service.GeofencingService.a
  at com.transistorsoft.locationmanager.service.GeofencingService.a
  at com.transistorsoft.locationmanager.service.GeofencingService$a$a.a
  at com.transistorsoft.locationmanager.service.GeofencingService$a$a.$r8$lambda$PJ5EoePOcmtk9xrfLgP7wtZLiok
  at com.transistorsoft.locationmanager.service.GeofencingService$a$a$$ExternalSyntheticLambda1.run
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1145)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:644)
  at java.lang.Thread.run (Thread.java:1012)
```

</details>

**Comment by christocracy** [comment]
>4.15.2

Update to latest version.  This was fixed in `4.15.3`

>[Android] Guard against Geofence SQLite query returning null in GeofencingService.

**Comment by mitul-neosoft** [comment]
That is a quick response.  will update the plugin and check.
Thanks @christocracy
